Arma3Helper.sh

```diff
@@ -130,26 +153,44 @@ elif [[ $1 = "winetricks" ]]; then
 		winetricks "${*:2}"
 	fi
 elif [[ $1 = "winecfg" ]]; then
-	echo "Starting winecfg for Arma's compatdata..."
-	echo
+	echo "Starting winecfg via winetricks for Arma's compatdata..."
+	_checkinstall "/usr/bin/winetricks" "winetricks"
 	export WINEPREFIX="$COMPAT_DATA_PATH/pfx"
 	winetricks winecfg
+# Updater
+elif [[ $1 = "update" ]]; then
+	echo -e "\e[31mUpdating the script will reset your changes in the script options!\e[0m"
+	echo "However, it will not reset your settings in ~/.arma3helper."
+	_confirmation "Are you sure?"
+	curl -o "$0" https://raw.githubusercontent.com/ninelore/armaonlinux/master/Arma3Helper.sh
+	echo "The Script was updated!"
+# create extermal config
+elif [[ $1 = "createconfig" ]]; then
+	if [[ -r "$USERCONFIG/config" ]]; then
```

Comment:
It's a little weird to check if a file is readable, and then ask if one wants to overwrite the file. I'd use `-f` for "is there a file at this path".
If you want to overdo it, then you can first check if the file exists and then if it's writable (`-w`) and bail out if it's not.

Reply:
I probably could check for writeable right away couldnt I? My intention was to not ask if the file is corrupted for whatever reason.

Reply:
The main thing I think you want to test here is if the file exists (`-e` or `-f`, it's debatable which one) so that user confirmation can be asked if needed.

The rest is just getting fancy, printing custom error messages about the file or directory not being writable, instead of letting curl spew its errors.

Reply:
changed to `-e`  in 3b1ac07